app/src/main/java/com/infomaniak/mail/data/models/correspondent/Recipient.kt

```diff
@@ -26,7 +28,11 @@ import kotlinx.serialization.Serializable
 
 @Parcelize
 @Serializable
-class Recipient : EmbeddedRealmObject, Correspondent {
+open class Recipient() : EmbeddedRealmObject, Correspondent {
+    constructor(email: String, name: String) : this() {
+        initLocalValues(email, name)
+    }
```

Comment:
If we want to go this way, we should totally remove `initLocalValues()`, and add an explanatory comment to insist on the fact that we need to use this secondary constructor.

This is a big change, it should be done on another PR, and this diff should be reverted.

Reply:
For a second, I believed I had no other option than using a constructor to do what I wanted and if I add the constructor I might as well use it everywhere. But actually it's not needed so yeah I can undo this